llvm/lib/Target/RISCV/RISCVExpandPseudoInsts.cpp

```diff
@@ -96,10 +98,96 @@ bool RISCVExpandPseudo::expandMI(MachineBasicBlock &MBB,
   case RISCV::PseudoVMSGE_VI:
   case RISCV::PseudoVMSGEU_VI:
     return expandCompareSelect(MBB, MBBI, NextMBBI);
+  case RISCV::PseudoVOR_VI_IMM11:
```

Comment:
This way works too
it is better to set flags in instrtuction, you can refer to `vlw12.v` instruction, I remember I set flags before
Ih this way, it is more friendly when later we need to support other instruction